modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -217,22 +216,74 @@ const OrderableTableRow = ({
 	);
 };
 
+const Table = ({
+	actions,
+	fields,
+	items,
+	onDragCrossover,
+	onDrop,
+	query,
+}: {
+	actions?: Array<IAction>;
+	fields: Array<IField>;
+	items: Array<any>;
+	onDragCrossover: Function;
+	onDrop: Function;
+	query: string;
+}) => {
+	const [, dropRef] = useDrop({
+		accept: ROW_DRAGGABLE,
+		drop() {
+			onDrop();
+		},
+	});
```

Comment:
We are adding table area as the drop area. I had to extract it to a separate component, because `react-dnd` requires `DndProvider` to be in external context.

Row area has a separate initialization as the draggable element, as well as temporary reorder on hover.